gitlens/gitlens-features.md

```diff
@@ -590,6 +638,18 @@ Or
 
 ***
 
+## Jira Integration
+
+The Jira Integration connected through your GitKraken Account gives you access to Jira [Autolinks](/gitlens/gitlens-features#autolinks) anywhere autolinks are supported in GitLens. Simply connect the integration and GitLens will automatically convert Jira IDs to links i.e. `ABC-123`.
+
+<img src="/wp-content/uploads/gl-jira-integration.png" class="img-bordered img-responsive center">
+
+To connect the integration, open the GitLens Sidebar, select `Cloud Integrations` from GitKraken Account, select Jira Cloud, and proceed with the integration connection by allowing GitKraken access to your Atlassian Account. You can also open gitkraken.dev/settings/integrations in a browser.
```

Comment:
```suggestion
To connect the integration, open the GitLens Sidebar, select `Cloud Integrations` from the GitKraken Account view, select Jira Cloud, and proceed with the integration connection by allowing GitKraken access to your Atlassian Account. You can also open gitkraken.dev/settings/integrations in a browser.
```

Reply:
By the way, if they do this:

> You can also open gitkraken.dev/settings/integrations in a browser

They will need to then hit the "refresh" icon on the GitKraken Account view for the integration to sync immediately to their account. That's because if they access these integration settings from outside of VSCode, GitLens doesn't know to check for their latest connections and sync until either the next regular check-in cycle (once every 24 hours) or the next time they refresh their account.